src/homebridge-vantage-static.ts

```diff
@@ -24,7 +28,7 @@ class VantageStaticPlatform implements StaticPlatformPlugin {
   private vantageController: VantageInfusionController;
   private interfaceSupportRequest: Array<Promise<void>>;
   private accessoriesDict: { [key: string]: AccessoryPlugin };
-  private vidNameMapping: { [key: string]: string };
+  private vidMapping: { [key: string]: { "Name"?: string, "Type"?: string } };
```

Comment:
is `Name` optional?

Reply:
yes. In case the default name from the configuration is suitable, but the type is not, you don't have to set `Name`